src/UCommerce.Transactions.Payments.Stripe/Configuration/Stripe.config

```diff
@@ -0,0 +1,16 @@
+﻿<configuration>
+	<components>
+		<!-- Stripe Payments -->
+		<component id="Stripe.com"
+		           service="Ucommerce.Transactions.Payments.IPaymentMethodService, Ucommerce"
+		           type="Ucommerce.Transactions.Payments.Stripe.StripePaymentMethodService, Ucommerce.Transactions.Payments.Stripe" />
```

Comment:
After forking the PR and building the produced dll name is `Ucommerce.Transactions.Payments.StripeNet` rather than `Ucommerce.Transactions.Payments.Stripe` this causes issues in the Ucommerce as It fails to resolve the type would it be possible to either change the project to StripeNet or the DLL itself?

Reply:
Cool, I'll make that a bit better - by using a "Stripe" namespace it originally conflicted with Stripe's own DLL.